### ISSUE TITLE
Grafana in Windows 11 Contrast Themes doesn't work well

### ISSUE THREAD

**Issue by imatwawana** [grafana/grafana#84095]
In Windows Settings turn on a Contrast Theme for a dark background with light text/elements in the foreground.  Then use MS Edge and a Grafana dashboard or generally browsing Grafana notice that some charts lines don’t appear visible at contrast levels.  Also in various places on the site I recall checkboxes that can’t be seen.

**Steps to reproduce**:

**Actual Result**:

**Expected Result**

**Relevant WCAG Criteria:** [#.#.# WCAG Criterion](link to https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0)

**Environment**:
- Grafana version:
- Data source type & version:
- User OS & Browser: Windows 11 & MS Edge
- Others:

**Comment by grafanabot** [comment]
@imatwawana please add one or more appropriate labels. Here are some tips:

- if you are making an issue, TODO, or reminder for yourself or your team, please add one label that best describes the product or feature area. Please also add the issue to your project board. :rocket:

- if you are making an issue for any other reason (docs typo, you found a bug, etc), please add at least one label that best describes the product or feature that you are discussing (e.g. `area/alerting`, `datasource/loki`, `type/docs`, `type/bug`, etc). [Our issue triage](https://github.com/grafana/grafana/blob/main/ISSUE_TRIAGE.md#3-categorizing-an-issue) doc also provides additional guidance on labeling. :rocket:

 Thank you! :heart:
